core/win32/events.mc

```diff
@@ -662,4 +662,12 @@ Language=English
 Application %1!s! (%2!s!). Out of contiguous memory. %3!s!
 .
 
+MessageId =
+Severity = Error
+Facility = DRCore
+SymbolicName = MSG_AVX_512_SUPPORT_INCOMPLETE
+Language=English
+Application %1!s! (%2!s!): AVX-512 was detected at PC 0x%3!lx!
```

Comment:
First, long is 32 bits in Visual Studio, so this the wrong size specifier.

Second, I don't know for sure whether anything but 's' works for Windows event log -- we had sort of been operating under that assumption as you can see from all of the other numeric params being 's' but I didn't write that code.

Reply:
That's why I initially didn't use the SYSLOG macro that goes through the gen_events script. Should we remove the @pc message then for now?

Reply:
Everywhere else we do a local snprintf to a buffer to solve this.

Reply:
Sure sure.. Done.